docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,17 +3,171 @@ title: Database Access AWS IAM Reference
 description: AWS IAM policies for Teleport database access.
 ---
 
+## Auto-discovery
 Teleport automatically discovers and configures IAM for RDS and Redshift given
```

Comment:
I know this is outside the changes in this PR, but I think "given proper IAM permissions" is throwing me off here, since it's not clear what this refers to until later. I also don't think we need the links to the two guides here, since we lay out the permissions below anyway. Could we remove this paragraph?

Reply:
What would you think of:

```markdown
With the appropriate IAM permissions, Teleport automatically discovers and configures
IAM policies for Amazon RDS and Redshift. 
```
Is there anything the two linked guides explain that the reader would be expected to know before reading this guide? Otherwise, it might be clearer not to mention them.

Reply:
That affects the permissions, no. The guides would only help the readers set the configuration file. But I agree that without the links, it is more clear. I'm updating it with your suggestion, thanks!